articles/how-to-fix-and-prevent-sql-injection-in-wordpress/index.md

```diff
@@ -0,0 +1,196 @@
+### Introduction
+SQL injection in [WordPress](https://kinsta.com/knowledgebase/what-is-wordpress/) is [ranked the second most critical vunerability](secure.wphackedhelp.com). [Updating your WordPress website](https://wordpress.org/support/article/updating-wordpress/) to the latest release shields you from SQL injection. Nevertheless, third party themes and plugins still pose a big threat to WordPress users. When you hear SQL injection, you understand it is the manipulation of a security flaw in database systems. WordPress as a platform runs server-side scripts in PHP. SQl and PHP hence make WordPress vulnerable to attackers via inserting malicious URLs embedded with commands that trigger behaviours from the database. Consequently,  sensitive information from your database is exposed or leaked to hackers which they can use to manipulate your WordPress website.
+
+#### What is SQLi?
+
+SQLi is an abbreviation for SQL injection.
+
+SQL injection is the injection or insertion of SQL code usually via data transmitted from a website. When an SQL injection attack is deployed successfully, it gives them access to sensitive information as well as ability to add/edit/delete data from a database. Generally, SQLi gives an attacker the possibility to execute all actions related to a database. SQLi takes advantage of PHP and SQl database hence posing a big risk to WordPress. Other than PHP, [the ASP language](https://www.w3schools.com/asp/default.ASP) is also prone to SQLi.
+
+An attacker makes use of data input by the user in to the database that is not masked sufficiently and that which contains meta characters i.e. semicolon, quotation marks and double hyphen. These parameters have specific functions with which the SQL interpreter uses to execute external commands. SQL injection targets old interfaces which often do not necessarily mask key parameters hence making ASP and PHP programs a perfect SQLi target for an attack.
```

Comment:
into
"isn't sufficiently masked and contains meta characters" would flow better.
semicolon and double hyphen should be plurals.